src/api/index.js

```diff
@@ -62,6 +66,9 @@ RippleAPI.prototype = {
   prepareTrustline,
   prepareOrder,
   prepareOrderCancellation,
+  prepareSuspendedPaymentCreate,
```

Comment:
Sadly, this might need to look like `prepareSuspendedPaymentCreation` for consistency